### ISSUE TITLE
TensorFlow Implementation 

### ISSUE THREAD

**Issue by sohamshashank1** [sararob/keras-wine-model#2]
Could you please provide the Tensorflow implementation of the same code, where input in Tokenized text, same as in the blog. But the layers defined using TF. 
Please

**Comment by sararob** [comment]
I'd like to add this. If you get to it before me feel free to submit a PR 😄

**Comment by Namansharma02** [comment]
I tried making this model but my accuracy is coming as zero.

Epoch 1/10
95646/95646 [==============================] - 63s 655us/step - loss: 1057.0534 - acc: 0.0000e+00

cant find the error

**Comment by Namansharma02** [comment]
is it because of the tensorflow version? mine is 1.12.0

**Comment by pgurazada** [comment]
@Namansharma02  In my case, running the model as per the blog post was giving `nan`'s in loss. Correcting for the missing values in the data frame amended the situation.  However, given that this is a regression problem, would the accuracy metric be of any value? 

One last suggestion - @sararob has already put up a refined implementation [here](https://colab.research.google.com/github/sararob/keras-wine-model/blob/master/keras-wide-deep.ipynb).
